pkg/proxy/iptables/proxier.go

```diff
@@ -967,6 +967,8 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 	}
 
+	serviceNoEndpointsTotalInternal := 0
```

Comment:
A comment here for the metric this is used for would be helpful

Reply:
Done